components/x-teaser/src/Image.jsx

```diff
@@ -32,6 +32,7 @@ export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, imageHighes
 	const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'))
 	const options = imageSize === 'XXL' && imageHighestQuality ? { quality: 'highest' } : {}
 	const imageSrc = useImageService ? imageService(image.url, ImageSizes[imageSize], options) : image.url
+	const alt = (image.altText || '').trim()
```

Comment:
I think it was in case the alt text that was passed was ' ' we wanted to use an empty string but tbh I don't fully remember. It was a Luke suggestion 😬

Reply:
Ok. It is no harm anyway. I was just curious 👍